core/embed/projects/bootloader/main.c

```diff
@@ -369,7 +369,7 @@ int bootloader_main(void) {
 #ifdef USE_TOUCH
   secbool touch_initialized = secfalse;
   secbool allow_touchless_mode = secfalse;
-#ifdef TREZOR_MODEL_T3T1
+#if defined TREZOR_MODEL_T3T1 || defined TREZOR_MODEL_T3W1
   // on T3T1, tester needs to run without touch, so making an exception
```

Comment:
maybe add T3W1 to the comment too?

Reply:
fixed https://github.com/trezor/trezor-firmware/pull/4350/commits/c927e916b21cd8a7ad65303ca8ddb0a8cb27c3e5